tracim/development.ini.base

```diff
@@ -38,6 +38,9 @@ cache_dir = %(here)s/data
 beaker.session.key = tracim
 beaker.session.secret = 3283411b-1904-4554-b0e1-883863b53080
 
+# Auth type
```

Comment:
A good practice : show in comments possible values. Here, it would be "internal" or "ldap"